manifests/i/InterfaceFluidics/Autobench/Beta/0.88.1.0/InterfaceFluidics.Autobench.Beta.installer.yaml

```
@@ -0,0 +1,18 @@
+# Created using wingetcreate 1.9.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: InterfaceFluidics.Autobench.Beta
+PackageVersion: 0.88.1.0
+Platform:
+- Windows.Universal
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+InstallerType: msix
+PackageFamilyName: Autobench-Beta_adgv3mcaarp50
+Installers:
+- Architecture: neutral
+  InstallerUrl: https://apps.interfacefluidics.com/autobench/beta/0.88.1.0/autobench-0.88.1.0.msix
+  InstallerSha256: E4BFFBE3C5A1FC290429AC88F232A59AC913C4E75D16777E41EBDB186DE224BF

```

```suggestion
  InstallerSha256: E2E8AC449B994492C3FF103EEE25A4BF82E6B323D0229A53E816502E6F87B61A
```

After making changes to the installer, update the hash.
If `SignatureSha256` has been changed, it also needs to be updated.

Hash by https://github.com/DuckDuckStudio/winget-tools/actions/runs/13547790244/job/37863610287#step:6:8